internal/manifest/coi_iso_tree.go

```diff
@@ -0,0 +1,127 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/osbuild/osbuild-composer/internal/disk"
+	"github.com/osbuild/osbuild-composer/internal/osbuild"
+	"github.com/osbuild/osbuild-composer/internal/users"
+)
+
+type COIISOTree struct {
```

Comment:
I lean towards thinking this should be CoreOSInstallerISOTree but that's a bit long. Maybe CoreOSISOTree?

Reply:
I like `CoreOSISOTree`.  Thanks.